functions/common/config-common/src/main/java/org/springframework/cloud/fn/common/config/SpelExpressionConverterConfiguration.java

```diff
@@ -38,7 +38,7 @@
 
 @Configuration
 @AutoConfigureAfter(name = "org.springframework.cloud.stream.config.SpelExpressionConverterConfiguration")
-@ConditionalOnMissingBean(type = "org.springframework.cloud.stream.config.SpelExpressionConverterConfiguration")
+@ConditionalOnMissingClass("org.springframework.cloud.stream.config.SpelExpressionConverterConfiguration")
```

Comment:
You said you have rebased to master.
What does this change do in this PR?

Nevertheless, I think this condition doesn't fit to the expectations.
We may have a class on CP, but exclude it from auto-configuration.
So, we will reject this config, but won't have any expected beans.

Reply:
The change that @dtur added originally did not work. So, he had to make that change. I just introduced it in the PR after talking to him.

Reply:
Yeah... OK. Let's see how it is going to live!
Anyway I think that `SpelExpressionConverterConfiguration` has to go from SCSt...